dumpgenerator.py

```diff
@@ -1200,7 +1206,7 @@ def getParameters(params=[]):
         sys.exit(1)
 
     # No download params and no meta info params? Exit
-    if (not args.xml and not args.images) and \
+    if (not args.xml and not args.images and args.resources == 'html') and \
```

Comment:
I kept html as the default though (the same as the current version which saves only index.html). This is because the other settings don't sanitise the IP address (yet).